### ISSUE TITLE
1.12.2 Schematic Format is different from 1.13+: Can't load

### ISSUE THREAD

**Issue by MrFaber** [Sloimayyy/mcschematic#5]
Hello, 

I need your help with a problem. 

I'm trying to load a mcschematic created schematic (output is set to: mcschematic.Version.JE_1_12_2) in minecraft 1.12.2 either with litematica, schematica or world edit mods. Neither of them can open mcschematic output file. 

World Edit in minecraft 1.16.5 and litematica in minecraft 1.16.5 having both no difficulties with the mcschematic output file. 

Assumption:
-world format ist different between 1.12.2 and 1.13+ (source: mojang)
-schematic format is different between 1.12.2 and 1.13+ (source: lunatrius, masady, sk89q)

proposed solution:
-state clearly in the documentation that 1.13+ schematic output is created, even if you put 1.12.2 in the version parameter. 
OR
-differentiate between old and new schematic format using the version parameter and create an 1.12 compatible output.

Thank you so much!

**Comment by Sloimayyy** [comment]
Hi! Thanks for the report!
Yeah the documentation is terrible. XD I been wanting to overhaul it for a bit now, but honestly I been lazy af XD

And you're right, MCSchem only handles sponge schematics format version 2, the one that's used in 1.13+, even though schematics are technically independent of Minecraft versions.

I honestly haven't been wanting to maintain the project so I'll probably just remove the 1.12 minecraft versions and under.

**Comment by MrFaber** [comment]
Heyho,

thank you so much for the incredibly qucik reply.

I spent the whole morning researching how to automatically create a statue
from a player skin and unfortunately there is no "cool" solution for 1.12.2
and below :/
Just for your comparison, thats what I had to do now: xD

Equip Bukkit 1.7.2 server with statuebuilder plugin and Worldedit. Build a
corresponding client.
Grab the appropriate skin and place the file in the plugin folder, create
the statue with a command, export with WorldEdit.
Load the exported schematic with Worldedit or Schematica in 1.12.2.
It's been a hell of a mess to figure that out and get the appropriate java
versions etc, but I'm glad that I at least have a solution now, I need it
for an LP series on Youtube.

thanks and best regards

Am Fr., 2. Feb. 2024 um 15:26 Uhr schrieb Sloimayyy <
***@***.***>:
